reacnetgenerator/utils.py

```
@@ -273,6 +276,264 @@ def bytestolist(x: bytes) -> Any:
+                os.unlink(path)
+            raise
+        self._paths[index] = path
+
+    def pop(self, index: int) -> Any:
+        """Read and remove one result from the spool."""
+        try:
+            path = self._paths.pop(index)
+        except KeyError as e:
+            raise RuntimeError("Ordered result is not available") from e
+        with open(path, "rb") as f:
+            value = pickle.load(f)
```

Ruff (0.16.1)

[error] `pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)